hyperion/renderer/html-template.js

```diff
@@ -72,7 +73,6 @@ export const getFooter = ({
       <script>window.__SERVER_STATE__=${serialize(state)}</script>
       <script>window.__DATA__=${serialize(data)}</script>
       <script defer="defer" type="text/javascript" src="https://cdn.polyfill.io/v2/polyfill.min.js"></script>
-      <script type="text/javascript" src="/static/js/bootstrap.js"></script>
```

Comment:
This needs to be there, at least in production!

Reply:
ok, sorry I forgot to remove that...